databasetest/docker.go

```diff
@@ -187,7 +164,7 @@ func (p *containerCloser) Close() error {
 		err = p.pool.Purge(p.container)
 	})
 	if err != nil {
-		return fmt.Errorf("failed stopping MySQL docker container: %w", err)
+		return fmt.Errorf("failed stopping dabase docker container: %w", err)
```

Comment:
"dabase"

Reply:
Its where information about District Attorneys is stored

Reply:
https://www.youtube.com/watch?v=gP3MuUTmXNk